openmc/material.py

```diff
@@ -287,12 +287,33 @@ def get_decay_photon_energy(
             units: str = 'Bq',
             volume: Optional[float] = None
         ) -> Optional[Univariate]:
-        r"""Return energy distribution of decay photons from unstable nuclides.
+        warnings.warn(
+            "The 'get_decay_photon_energy' property has been replaced by the "
+            "get_decay_energy_distribution() method and will be removed in a future "
+            "version.", FutureWarning)
+        return self.get_decay_energy_distribution(
+            radiation_type = 'photon', # FIND DICTIONARY ENTRIES
```

Comment:
```suggestion
            radiation_type = 'photon',
```
Just wondering if we need the inline comment here